common/src/wh/components/navbar/components.cljc

```diff
@@ -36,6 +39,42 @@
       ^{:key (or path route)}
       [dropdown-element el opts])]))
 
+(defn search-dropdown-element [text]
+  [:li (merge (util/smc styles/dropdown__element))
+   [:div (util/smc styles/dropdown__search__option)
+    [:a (merge {
+          :on-click #((dispatch [:wh.search/search-with-value text]))
+          :class (util/mc styles/dropdown__link__text)
+        })
+        (let [search-value (or (<sub [::subs/search-value]) "")]
+            [:div {:dangerouslySetInnerHTML {:__html 
+              (clojure.string/replace text (re-pattern (str "(?i)" search-value)) #(str "<b>" %1 "</b>"))
+            }}]
+        )
+    ]
+    [icon "close" :on-click #(
+      (fn [e] (
+        (let [local-search (<sub [::subs/local-search])]
+        (let [values (remove (fn [x] (= x text)) local-search)]
+          (.setItem js/localStorage "local_search" 
+            (clojure.string/join "||" values) 
+          )
+          (dispatch [:wh.components.navbar.events/set-local-search values])
+        ))
+      ))
+    )]
+    
+  ]])
```

Comment:
This style, of breaking out the parentheses, is not idiomatic and not inline with the style of the rest of the code base. Please condense this. Thanks

Reply:
@acron0 fixed those and also added background color to entered value 
![image](https://user-images.githubusercontent.com/15967809/97114035-2572d700-1714-11eb-8033-6a8f36c02e43.png)